docs/root/configuration/http/http_filters/proto_message_extraction_filter.rst

```diff
@@ -64,21 +64,24 @@ Output Format
 The extracted requests and responses will be  will be added in the dynamic
 ``metadata<google.protobuf.Struct>`` with the same layout of the message.
 
-For the default `FIRST_AND_LAST` mode, the output will be like:
+For the default ``FIRST_AND_LAST`` mode, the output will be like:
 
 Case: Non-Streaming requests/response
 
 .. code-block:: json
+   :force:
 
   {
     "requests":{
        "first":{
           "foo": "val_foo1",
+          ...
```

Comment:
wasnt sure if other fields are possible/needed here - but saw the `...` was removed while fixing the markup on the api instances

Reply:
More fields are possible after "foo".

`...` seems fine to me.

I am assuming ` :force:` is added because of `...`.

Reply:
yeah - it allows it to render not-quite json